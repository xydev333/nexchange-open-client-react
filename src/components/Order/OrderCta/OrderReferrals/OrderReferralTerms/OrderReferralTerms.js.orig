import React from 'react';
import { I18n } from 'react-i18next';
import { Modal } from 'react-bootstrap';

<<<<<<< HEAD:src/components/Order/OrderCta/OrderReferrals/OrderReferralTerms/OrderReferralTerms.js
const OrderReferralTerms = props => (
  <Modal id="OrderReferralTerms" show={props.show} onHide={props.onClose}>
=======
const ReferralTerms = props => (
  <I18n ns="translations">
   {(t) => (
  <Modal id="ReferralTerms" show={props.show} onHide={props.onClose}>
>>>>>>> Start merging i18n integration:src/components/ReferralTerms.js
    <div className="modal-content">
      <div className="modal-header">
        <button type="button" className="close" data-dismiss="modal" aria-hidden="true" onClick={props.onClose}>
          <i className="material-icons">clear</i>
        </button>
        <h4 className="modal-title">{t('referral.3')}</h4>
      </div>

      <div className="modal-body">
        <div className="row">
          <div className="col-xs-12">
<<<<<<< HEAD:src/components/Order/OrderCta/OrderReferrals/OrderReferralTerms/OrderReferralTerms.js
            <p>
              {' '}
              You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by
              time! For more details, and further assistance, please contact support via our Live chat!
            </p>
=======
            <p> {t('referral.4')}</p>
>>>>>>> Start merging i18n integration:src/components/ReferralTerms.js
          </div>
        </div>
      </div>
    </div>
  </Modal>
   )}
  </I18n>
);

export default OrderReferralTerms;
