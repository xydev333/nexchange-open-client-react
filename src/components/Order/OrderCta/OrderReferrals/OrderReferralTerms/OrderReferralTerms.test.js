import React from 'react';
import { shallow } from 'enzyme';
import OrderReferralTerms from './OrderReferralTerms';

describe('OrderReferralTerms', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderReferralTerms />)).toMatchSnapshot();
  });
});
