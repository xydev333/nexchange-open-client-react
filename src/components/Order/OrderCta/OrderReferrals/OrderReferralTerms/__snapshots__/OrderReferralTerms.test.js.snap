// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderReferralTerms renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderReferralTerms />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "animation": true,
      "autoFocus": true,
      "backdrop": true,
      "bsClass": "modal",
      "children": <div
        className="modal-content"
>
        <div
                className="modal-header"
        >
                <button
                        aria-hidden="true"
                        className="close"
                        data-dismiss="modal"
                        onClick={undefined}
                        type="button"
                >
                        <i
                                className="material-icons"
                        >
                                clear
                        </i>
                </button>
                <h4
                        className="modal-title"
                >
                        Terms & Conditions
                </h4>
        </div>
        <div
                className="modal-body"
        >
                <div
                        className="row"
                >
                        <div
                                className="col-xs-12"
                        >
                                <p>
                                         
                                        You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                                </p>
                        </div>
                </div>
        </div>
</div>,
      "dialogComponentClass": [Function],
      "enforceFocus": true,
      "id": "OrderReferralTerms",
      "keyboard": true,
      "manager": ModalManager {
        "add": [Function],
        "containers": Array [],
        "data": Array [],
        "handleContainerOverflow": true,
        "hideSiblingNodes": true,
        "isTopModal": [Function],
        "modals": Array [],
        "remove": [Function],
      },
      "onHide": [Function],
      "renderBackdrop": [Function],
      "restoreFocus": true,
      "show": false,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="modal-header"
>
            <button
                        aria-hidden="true"
                        className="close"
                        data-dismiss="modal"
                        onClick={undefined}
                        type="button"
            >
                        <i
                                    className="material-icons"
                        >
                                    clear
                        </i>
            </button>
            <h4
                        className="modal-title"
            >
                        Terms & Conditions
            </h4>
</div>,
          <div
            className="modal-body"
>
            <div
                        className="row"
            >
                        <div
                                    className="col-xs-12"
                        >
                                    <p>
                                                 
                                                You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                                    </p>
                        </div>
            </div>
</div>,
        ],
        "className": "modal-content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                aria-hidden="true"
                className="close"
                data-dismiss="modal"
                onClick={undefined}
                type="button"
>
                <i
                                className="material-icons"
                >
                                clear
                </i>
</button>,
              <h4
                className="modal-title"
>
                Terms & Conditions
</h4>,
            ],
            "className": "modal-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "aria-hidden": "true",
                "children": <i
                  className="material-icons"
>
                  clear
</i>,
                "className": "close",
                "data-dismiss": "modal",
                "onClick": undefined,
                "type": "button",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "clear",
                  "className": "material-icons",
                },
                "ref": null,
                "rendered": "clear",
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Terms & Conditions",
                "className": "modal-title",
              },
              "ref": null,
              "rendered": "Terms & Conditions",
              "type": "h4",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
>
              <div
                            className="col-xs-12"
              >
                            <p>
                                           
                                          You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                            </p>
              </div>
</div>,
            "className": "modal-body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="col-xs-12"
>
                <p>
                                 
                                You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                </p>
</div>,
              "className": "row",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p>
                   
                  You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
</p>,
                "className": "col-xs-12",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!",
                ],
                "type": "p",
              },
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "animation": true,
        "autoFocus": true,
        "backdrop": true,
        "bsClass": "modal",
        "children": <div
          className="modal-content"
>
          <div
                    className="modal-header"
          >
                    <button
                              aria-hidden="true"
                              className="close"
                              data-dismiss="modal"
                              onClick={undefined}
                              type="button"
                    >
                              <i
                                        className="material-icons"
                              >
                                        clear
                              </i>
                    </button>
                    <h4
                              className="modal-title"
                    >
                              Terms & Conditions
                    </h4>
          </div>
          <div
                    className="modal-body"
          >
                    <div
                              className="row"
                    >
                              <div
                                        className="col-xs-12"
                              >
                                        <p>
                                                   
                                                  You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                                        </p>
                              </div>
                    </div>
          </div>
</div>,
        "dialogComponentClass": [Function],
        "enforceFocus": true,
        "id": "OrderReferralTerms",
        "keyboard": true,
        "manager": ModalManager {
          "add": [Function],
          "containers": Array [],
          "data": Array [],
          "handleContainerOverflow": true,
          "hideSiblingNodes": true,
          "isTopModal": [Function],
          "modals": Array [],
          "remove": [Function],
        },
        "onHide": [Function],
        "renderBackdrop": [Function],
        "restoreFocus": true,
        "show": false,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="modal-header"
>
              <button
                            aria-hidden="true"
                            className="close"
                            data-dismiss="modal"
                            onClick={undefined}
                            type="button"
              >
                            <i
                                          className="material-icons"
                            >
                                          clear
                            </i>
              </button>
              <h4
                            className="modal-title"
              >
                            Terms & Conditions
              </h4>
</div>,
            <div
              className="modal-body"
>
              <div
                            className="row"
              >
                            <div
                                          className="col-xs-12"
                            >
                                          <p>
                                                         
                                                        You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                                          </p>
                            </div>
              </div>
</div>,
          ],
          "className": "modal-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <button
                  aria-hidden="true"
                  className="close"
                  data-dismiss="modal"
                  onClick={undefined}
                  type="button"
>
                  <i
                                    className="material-icons"
                  >
                                    clear
                  </i>
</button>,
                <h4
                  className="modal-title"
>
                  Terms & Conditions
</h4>,
              ],
              "className": "modal-header",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "aria-hidden": "true",
                  "children": <i
                    className="material-icons"
>
                    clear
</i>,
                  "className": "close",
                  "data-dismiss": "modal",
                  "onClick": undefined,
                  "type": "button",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "clear",
                    "className": "material-icons",
                  },
                  "ref": null,
                  "rendered": "clear",
                  "type": "i",
                },
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Terms & Conditions",
                  "className": "modal-title",
                },
                "ref": null,
                "rendered": "Terms & Conditions",
                "type": "h4",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="row"
>
                <div
                                className="col-xs-12"
                >
                                <p>
                                                 
                                                You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                                </p>
                </div>
</div>,
              "className": "modal-body",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="col-xs-12"
>
                  <p>
                                     
                                    You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
                  </p>
</div>,
                "className": "row",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                     
                    You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!
</p>,
                  "className": "col-xs-12",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "You will receive 50% of our revenues (0.25% of the turnover) of all of your referrals. This offer is currently not limited by time! For more details, and further assistance, please contact support via our Live chat!",
                  ],
                  "type": "p",
                },
                "type": "div",
              },
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
